package com.github.soup.group.exception

class GroupNameAlreadyExistException : RuntimeException(
    "그룹 이름이 이미 존재합니다."
)